packages/headless/src/features/triggers/triggers-slice-functions.test.ts

```diff
@@ -0,0 +1,206 @@
+import {Trigger} from '../../api/common/trigger';
```

Comment:
The commerce and non-commerce triggers reducers share the exact same logic, so I think it makes sense to extract this logic into functions, test those functions thoroughly, call those functions in the reducers, and test that those functions are getting called in the reducers.

Reply:
(this may not be a pattern we want to adopt widely, but I wanted to try it out)